src/ra_utility.h

```diff
@@ -10,23 +10,17 @@
    as it will be auto-deduced unless stated otherwise.
 */
 namespace ra {
-#pragma warning(push)
 
-#pragma warning(push)
-// TODO: Finish narrow_cast in another PR
-// we don't use gsl::narrow_cast, ra::narrow_cast won't trigger this when its
-// done
-#pragma warning(disable : 26472)
 template<typename SignedType, typename = std::enable_if_t<std::is_signed_v<SignedType>>>
 _NODISCARD _CONSTANT_FN to_unsigned(_In_ SignedType st) noexcept
 {
-    return static_cast<std::make_unsigned_t<SignedType>>(st);
+    return gsl::narrow_cast<std::make_unsigned_t<SignedType>>(st);
```

Comment:
we gotta use gsl::narrow_cast instead of gsl::narrow because gsl::narrow throws (meaning not a constant expression).